tests/foreman/ui/test_package.py

```diff
@@ -68,6 +68,72 @@ def module_rh_repo(module_sca_manifest_org, module_target_sat):
     return module_target_sat.api.Repository(id=repo_id).read()
 
 
+@pytest.fixture(scope='module')
+def module_rhel8_repo(module_sca_manifest_org, module_target_sat):
+    "Enable and sync rhel8 appstream repository"
+    return module_target_sat.cli_factory.setup_org_for_a_rh_repo(
+        {
+            'product': PRDS['rhel8'],
+            'repository-set': REPOSET['rhel8_aps'],
+            'repository': REPOS['rhel8_aps']['name'],
+            'organization-id': module_sca_manifest_org.id,
+            'releasever': REPOS['rhel8_aps']['releasever'],
+        },
+        force_use_cdn=True,
+    )
+
+
+@pytest.mark.rhel_ver_match('8')
```

Comment:
@vijaysawant Is there a specific reason for using RHEL 8 here (I couldn't find it specified anywhere in the bug itself)? Can't we for example use `@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])`

Reply:
I believe httpd package rpm is part of rhel 8 appstream, so I used rhel 8 and enabled appstream repository.

Reply:
All right then.